src/Orleans/Utils/SharedState/ISharedStatePublisher.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+namespace Orleans
```

Comment:
classes in ../Utils/SharedState/.. are shared state infrastructure.  Common to all uses of the shared state.